api/pkg/client/mlflow/mlflow.go

```diff
@@ -13,8 +13,6 @@ import (
 )
 
 type Service interface {
-	searchRunsForExperiment(ExperimentID string) (SearchRunsResponse, error)
-	searchRunData(RunID string) (SearchRunResponse, error)
```

Comment:
Woops my bad I didn't catch this earlier ahhaha

Reply:
no worries 😁